### PR TITLE
create hilaryparker2

### PR DESCRIPTION
**Pull request by hilaryparker**
doing it right this time

@githubteacher please review!

### CODE REVIEW

**Comment by hilaryparker** [comment]
@githubteacher 
@tinywizard hi!

**Comment by tinywizard** [comment]
@hilaryparker nice work!!

File: docs/hilaryparker2 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Hilary Parker
+
+Data analyst at Etsy
```
Comment (tinywizard) [inline docs/hilaryparker2]: Plz add line referring to how you're an awesome person.  Thx.

**Comment by rfriedel** [comment]
Please merge your changes ! :+1:

File: docs/hilaryparker2 (hunk 1 above)
Comment (hilaryparker) [inline docs/hilaryparker2]: :smile:

Comment (toriagibbs-zz) [inline docs/hilaryparker2]: Etsy people! Helloooo!

Comment (hilaryparker) [inline docs/hilaryparker2]: hi @toriagibbs!

**Comment by githubteacher** [comment]
:ship: this feature!

**Comment by githubteacher** [comment]
:shipit:

**Comment by hilaryparker** [comment]
@githubteacher ty for the shoutout, my absolute favorite mug

**Pull request merged**